### DISCUSSION TITLE
Difference between image and logo in quarto website yaml

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#986] [Q&A] [answered]
What is the difference in the quarto web yaml between image and logo?

**Comment by jmbuhr** [comment, marked as answer]
The image is used as a preview image if you share a link to your quarto site:
https://quarto.org/docs/reference/formats/html.html#website
The logo is shown in the navigation bar on your site: https://quarto.org/docs/websites/website-navigation.html#top-navigation
